frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -190,6 +190,15 @@ entitydetails-Helpers--locales = Locales
 entitydetails-Metadata--comment =
     .title = Comment
 
+entitydetails-Metadata--group_commment =
```

Comment:
Please don't use underscores in l10n IDs but dashes instead. Here it should be `entitydetails-Metadata--group-comment` instead (note also that you put 3 "m" in "comment" :wink: ).